src/simplewallet/simplewallet.cpp

```diff
@@ -1676,7 +1676,7 @@ void simple_wallet::on_money_received(uint64_t height, const crypto::hash &txid,
     m_refresh_progress_reporter.update(height, true);
 }
 //----------------------------------------------------------------------------------------------------
-void simple_wallet::on_unconfirmed_money_received(uint64_t height, const crypto::hash &txid, const cryptonote::transaction& tx, uint64_t amount)
+void simple_wallet::on_unconfirmed_money_received(uint64_t height, const crypto::hash &txid, uint64_t amount)
```

Comment:
related to https://github.com/monero-project/monero/pull/2239
wallet.cpp didn't have the tx argument. I removed it from wallet2 and simplewallet for them to match.